spec/features/users_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+feature "User creates an account" do
+  scenario "with a valid username, email and password" do
+    visit new_user_path
+
+    fill_in "Username", with: "felixpip"
```

Comment:
Use field id or name attribute instead of label.
Alternatively, use translations for the label.

Reply:
Fixed up commit (263fba0) in `--fixup` in sha: d234d15
Will add translations in a seperate feature branch 👍